app/genesis.go

```diff
@@ -170,20 +136,21 @@ func NewDefaultGenesisState() GenesisState {
 		},
 		SlashingData: slashing.GenesisState{
 			Params: slashing.Params{
-				MaxEvidenceAge:          defaultUnbondingTime,
 				SignedBlocksWindow:      60 * 4, // ~20min
-				DowntimeJailDuration:    0,
+				DowntimeJailDuration:    1,
 				MinSignedPerWindow:      sdk.NewDecWithPrec(80, 2),         // 80%
 				SlashFractionDoubleSign: sdk.NewDecWithPrec(5, 2),          // 5%
 				SlashFractionDowntime:   sdk.NewDec(5).Quo(sdk.NewDec(10000)), // 0.05%
 			},
 		},
 		DistrData: distr.GenesisState{
 			FeePool:             distr.InitialFeePool(),
-			CommunityTax:        sdk.NewDecWithPrec(10, 2), // 10%
-			BaseProposerReward:  sdk.NewDecWithPrec(1, 2),  // 1%
-			BonusProposerReward: sdk.NewDecWithPrec(5, 2),  // 5%
-			WithdrawAddrEnabled: true,
+			Params:              distr.Params{
+				CommunityTax:        sdk.NewDecWithPrec(10, 2), // 10%
```

Comment:
Are we sure that we will start from non zero community tax?

Reply:
It's for local development testnets gen